config/routes.rb

```diff
@@ -20,10 +20,14 @@
   # Account home page
   scope module: :account do
     resources :identities, except: [:show] do
-      get 'mernis', on: :member
+      collection do 
```

Comment:
Burada satır sonunda satırı seçtiğinde görebileceğin bir boşluk karakteri var! Sublime kullanıyorsan acilen kurmalısın: https://github.com/SublimeText/TrailingSpaces. Başka editör kullanıyorsan da satır sonu boşluklarını gördüğünden bir şekilde emin olmalısın.

Tek tek yazmıyorum, toplamda 4 yerde bu durum var `collection do ` ve bu blokları kapatan `end `lerde.